dags/openshift_nightlies/util/manifest.py

```diff
@@ -15,23 +18,36 @@ def __init__(self, root_dag_dir):
         self.release_stream_base_url = var_loader.get_secret("release_stream_base_url")
         self.get_latest_releases()
 
-    def get_latest_releases(self): 
+    def get_latest_releases(self):
         release_streams = [ version['releaseStream'] for version in self.yaml['versions']]
+        print(release_streams)
         self.latest_releases = {}
         for stream in release_streams:
-            url = f"{self.release_stream_base_url}/{stream}/latest"
-            response  = requests.get(url)
-            if response.status_code != 200: 
-                raise Exception(f"Can't get latest release from OpenShift Release API, API Returned {response.status_code}, {url}")
+            for arch in [ self.AMD64 , self.ARM64 ]:
+                if arch == self.ARM64:
+                    base_url_arm = self.release_stream_base_url.replace("openshift-release",f"openshift-release-{self.ARM64}")
```

Comment:
no i got that part about the different binaries, i was wondering if we could just use an if condition rather than if inside a for, but i figured that won't work. got it, thanks.

Reply:
Oh, you're right. the loop is not required and doesn't add anything meaningful.  I removed it and tests still pass. Thanks for pointing that out.